test/unit/rubygem_test.rb

```diff
@@ -381,10 +381,6 @@ class RubygemTest < ActiveSupport::TestCase
       end
     end
 
-    should "return current version" do
-      assert_equal @rubygem.versions.first, @rubygem.versions.most_recent
-    end
-
```

Comment:
I removed this test because it is testing for versions without creating any version in setup (both `actual` and `expected` values are `nil`). I stopped tracking where did this test went wrong after [I reached 2009](https://github.com/rubygems/rubygems.org/commit/bbadbea3f363f91055c8fe37677fe517793f4390#diff-ee96e2d5cc5ac47f847e19de030867cdR57). There is no good reason why `versions.first` and `versions.most_recent` should be equal anyway.
As of testing *current version* or *most recent versions* is concerned, we have already done it [here](https://github.com/rubygems/rubygems.org/blob/master/test/unit/rubygem_test.rb#L57-L164).